File: app/src/main/java/com/redteamobile/smart/util/FileUtil.java
package com.redteamobile.smart.util;

import android.content.Context;
import android.util.Log;

import com.redteamobile.smart.Constant;

import java.io.File;

public class FileUtil {

    /**
     * @param context
     * @return 存放文件地方，
     */
    public static String getAppPath(Context context) {
        Log.e("TAG", "getAppPath: "+ context.getFilesDir().getParent() );
        return context.getFilesDir().getParent();
    }
}

File: app/src/main/java/com/redteamobile/smart/agent/util/FileUtil.kt
package com.redteamobile.smart.agent.util

import android.content.Context
import com.redteamobile.smart.Constant
import java.io.File

class FileUtil {

    companion object {
        private val TAG = "FileUtil"

        fun getFilesDir(context: Context): File {
            var cacheDir: File? = File(Constant.FILE_STORAGE_PATH)
            if (cacheDir == null) {
                cacheDir = File(Constant.FILE_STORAGE_PATH)
            }
            return cacheDir
        }
    }
}
